### PR TITLE
update reference values for translation

### PR DESCRIPTION
**Pull request by nonew-star**

### CODE REVIEW

File: resources/language/Swedish/strings.po [hunk 1]
```diff
@@ -197,23 +197,23 @@ msgstr "Upplösningsinställning för TV-serier"
...
 
 msgctxt "#30051"
 msgid "Share ratio limit, percentage"
-msgstr "Gräns för delningskvot (procent)"
+msgstr "Gräns för delningskvot, procent"
 
 msgctxt "#30052"
 msgid "Seed time ratio limit, percentage"
-msgstr "Kvotgräns för distributionstid (procent)"
+msgstr "Kvotgräns för delningstid (procent)"
```
Comment (antonsoroko) [inline resources/language/Swedish/strings.po:212]: maybe also `, procent` for consistency

File: resources/language/Swedish/strings.po [hunk 2]
```diff
@@ -8,7 +8,7 @@ msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2016-01-27 19:53+0200\n"
-"PO-Revision-Date: 2021-03-11 12:31+0100\n"
+"PO-Revision-Date: 2021-05-23 10:56+0200\n"
 "Last-Translator: \n"
```
Comment (antonsoroko) [inline resources/language/Swedish/strings.po:12]: also if you like you can put your name/nickname here.
e.g. `"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"`

**Comment by elgatito** [comment]
@nonew-star Is this finished?

**Comment by nonew-star** [comment]
@elgatito Yes

**Comment by elgatito** [comment]
@nonew-star Thanks. Accepting.

**Pull request merged**